client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -94,9 +94,34 @@ export const UserProfilePage = () => {
             loadingText={'Saving Profile Changes'}
             submitText={'Save Profile Changes'}
           />
-          <Button colorScheme={'red'} marginBlock={'2em'} onClick={clickDelete}>
-            Delete My Data
-          </Button>
+          <Flex gap="2em" marginBlock={'2em'}>
+            <Button
+              height={'2.8em'}
+              paddingBlock={'.65em'}
+              paddingInline={'.4em'}
+              colorScheme={'red'}
+              onClick={clickDelete}
+            >
+              Delete My Data
+            </Button>
+            <Link
+              fontWeight="600"
+              background={'gray.85'}
+              color={'gray.10'}
+              height={'100%'}
+              size={'lg'}
+              borderRadius={'5px'}
+              paddingBlock={'.65em'}
+              paddingInline={'.4em'}
+              _hover={{ color: 'gray.85', backgroundColor: 'gray.10' }}
+              download={`${userInfo.name}.json`}
+              href={`data:text/json;charset=utf-8,${encodeURIComponent(
+                JSON.stringify(userInfo),
+              )}`}
+            >
+              Download your data
```

Comment:
interesting, although `area` has `download` and `href` attributes, and should be doing the same thing, it won't be rendered in React. Don't know why tho

Reply:
`area` needs to be used with `map`. Why `area`, though?